public/styles/module.css

```
@@ -162,3 +162,354 @@ summary.collapsibleHeader {
+    text-transform: uppercase;
+    letter-spacing: 0.05em;
+    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+}
+
+.creatureBuilderFormModern .preview-roadmap-level {
+    display: block;
+    padding: 2px 6px 4px;
+    margin-bottom: 2px;
+    font-size: 12px;
+    font-weight: 700;
+    word-wrap: break-word;
```

Stylelint (17.7.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)